### PR TITLE
[FUSEQE-1111] Salesforce producers

### PR DESCRIPTION
**Pull request by avano**
//test: @integration-sf-producers

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Can one of the admins verify this patch?

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: rest-tests/src/test/resources/features/integration-sf-producers.feature [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    Then wait for integration with name: "AMQ-SF-AMQ fetch record" to become active
+    When publish message with content '{"Id":"LEAD_ID"}' to queue "sf-producers-input"
+    Then verify that lead json object was received from queue "sf-producers-output"
+
+  @integration-sf-producers-fetch-record-external-id
+  Scenario: Fetch record with external id
+    When create SF "get-sobject-with-id" action step with properties
+      | sObjectName   | Lead  |
+      | sObjectIdName | Email |
+    And create AMQ "publish" action step with destination type "queue" and destination name "sf-producers-output"
```
Comment (mcada) [inline rest-tests/src/test/resources/features/integration-sf-producers.feature]: We should be consistent with indentation in whole feature file

Comment (avano) [reply rest-tests/src/test/resources/features/integration-sf-producers.feature]: :+1:

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by avano** [comment]
retest this please

**Pull request merged**